api/internal/target/kusttarget_configplugin.go

```diff
@@ -27,8 +30,28 @@ import (
 // image tag transforms.  In these cases, we'll need
 // N plugin instances with differing configurations.
 
-func (kt *KustTarget) configureBuiltinGenerators() (
-	result []resmap.Generator, err error) {
+var builtinKinds = map[builtinhelpers.BuiltinPluginType]string{
```

Comment:
We should try to avoid manually defining this relationship. We already have the inverse of this map in builtins.go. How about we modify the init function there to create this version as well?

Reply:
Thanks for pointing this out, while taking your suggestion I realized I can just use the `BuiltinPluginType.String()` method, so no need to define this relationship.